src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -23,7 +24,7 @@
 	);
 </script>
 
-<TokensSkeletons>
+<TokensSkeletons rows={ERC20_CONTRACTS.length + 1}>
```

Comment:
Actually I would set a smaller value. Maybe 3 or 5 too.

See for example this current screenshot of the loading experience:

<img width="1536" alt="Capture d’écran 2024-08-12 à 08 58 46" src="https://github.com/user-attachments/assets/1d1e3d83-bf57-4960-a191-b5ed3cba4a1e">

WDYT?

Reply:
ah yes, for sure, then I just default both of them to 5, it seems a good middle ground for both


<img width="1106" alt="Screenshot 2024-08-12 at 09 12 09" src="https://github.com/user-attachments/assets/1565d09b-4809-4884-8665-ac941c108af7">
<img width="291" alt="Screenshot 2024-08-12 at 09 12 24" src="https://github.com/user-attachments/assets/d2d7517b-58a9-4ee3-b517-78ee441322b5">
<img width="1106" alt="Screenshot 2024-08-12 at 09 12 44" src="https://github.com/user-attachments/assets/664d19d8-f403-41ce-a4b0-545b8589b4dc">
<img width="1104" alt="Screenshot 2024-08-12 at 09 12 53" src="https://github.com/user-attachments/assets/9414d651-913a-456d-9775-473a56d27677">

Reply:
Sounds good

Reply:
ready for another review